### DISCUSSION TITLE
onPressIn and onPressOut not working

### DISCUSSION THREAD

**Discussion by aniketgupta8960** [react-native-tvos/react-native-tvos#346] [Q&A] [answered]
### Description

I am trying to build a mouse pointer in tvOs which will move in all direction.Currently I am using the useTVEventHandler and it gives callback for swipeLeft or swipeRight but I wanted continuous callback till the time user doesn't remove his finger from the appletv remote just like touchesBegan and touchesMoved in swift. Is there a way to implement these two function in tvOS.If yes please suggest a way to implement this.

### Version

npm:react-native-tvos@0.68.2-2

### Output of `npx react-native info`

System:
    OS: macOS 12.3.1
    CPU: (8) x64 Intel(R) Core(TM) i5-1038NG7 CPU @ 2.00GHz
    Memory: 36.59 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 18.0.0 - /usr/local/bin/node
    Yarn: 1.22.10 - ~/‘~/.npm-global/bin/yarn
    npm: 8.5.5 - /usr/local/bin/npm
    Watchman: 2022.03.21.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    Android SDK:
      API Levels: 21, 23, 25, 27, 28, 29, 30, 31, 33
      Build Tools: 28.0.3, 29.0.2, 29.0.3, 30.0.2, 30.0.3, 31.0.0
      System Images: android-28 | Google Play Intel x86 Atom, android-29 | Intel x86 Atom_64, android-29 | Google APIs Intel x86 Atom, android-30 | Google TV Intel x86 Atom, android-30 | Google APIs Intel x86 Atom
      Android NDK: 22.1.7171670
  IDEs:
    Android Studio: 4.1 AI-201.8743.12.41.6953283
    Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.12 - /usr/local/opt/openjdk@11/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2 
    react-native: Not Found
    react-native-macos: Not Found
    react-native-tvos:  0.68.2-2 
  npmGlobalPackages:
    *react-native*: Not Found

### Steps to reproduce

1. Create a touchable component and try to move left and right.To move that component you will have to remove your finger every-time then only the cursor will move.

**Comment by leonchabbey** [comment, marked as answer]
You can toggle a pan gesture listener with `TVEventControl.enableTVPanGesture();` and `TVEventControl.disableTVPanGesture();`.

You will then receive in your `useTVEventHandler` an event with  `eventType: "pan"` and a body containing:
```
state: "Began" | "Changed" | "Ended",
x: number,
y: number,
velocityx: number,
velocityy: number
```
